htdocs/comm/action/card.php

```diff
@@ -393,31 +393,36 @@
 				$moreparam = '';
 				if ($user->id != $object->userownerid) $moreparam = "filtert=-1"; // We force to remove filter so created record is visible when going back to per user view.
 
-                //Create eminder
+                //Create reminder
                 if ($addreminder == 'on'){
                     $actionCommReminder = new ActionCommReminder($db);
 
-                    $dateremind = dol_time_plus_duree($datep, -$offsetvalue, 'i');
+                    $dateremind = dol_time_plus_duree($datep, -$offsetvalue, $offsetunit);
 
                     $actionCommReminder->dateremind = $dateremind;
                     $actionCommReminder->typeremind = $remindertype;
-                    $actionCommReminder->fk_user = $user;
                     $actionCommReminder->offsetunit = $offsetunit;
                     $actionCommReminder->offsetvalue = $offsetvalue;
                     $actionCommReminder->status = $actionCommReminder::STATUS_TODO;
                     $actionCommReminder->fk_actioncomm = $object->id;
                     if ($remindertype == 'email') $actionCommReminder->fk_email_template = $modelmail;
 
-                    $res = $actionCommReminder->create($user);
-
-                    if ($res <= 0){
-                        // If error
-                        $error++;
-                        $langs->load("errors");
-                        $error = $langs->trans('ErrorReminderActionCommCreation').' '.$actionCommReminder->error;
-                        setEventMessages($error, $actionCommReminder->errors, 'errors');
-                        $action = 'create'; $donotclearsession = 1;
-                    }
+                    // the notification must be created for every user assigned to the event
+					foreach ($object->userassigned as $userassigned)
```

Comment:
I dont't think it is the decision of the one who create an event and its invitation to decide for each users if and when they must receive a notification. It will be interesting to know how google works ? If you create 1 event with 2 attendee. And you set remind to 30mn before. Does the attendee also have a setup of 30mn before ?

Reply:
in this case, google notifies everyone at the same time